jekyll/_cci2/optimization-cookbook.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: classic-docs
+title: "CircleCI Optimizations Cookbook"
+short-title: "Optimizations Cookbook"
+description: "Starting point for Optimizations Cookbook"
+categories: [getting-started]
+order: 1
+---
+
+The *CircleCI Optimizations Cookbook* is a collection of individual use cases (referred to as "recipes") that provide you with detailed, step-by-step instructions on how to perform various optimization tasks using CircleCI resources. This guide, and it associated sections, will enable you to quickly and easily perform easily repeatable optimization tasks on the CircleCI platform.
+
+* TOC
+{:toc}
+
+## Introduction
+
+When work with the CircleCI platform, one of the most critical tasks you will want to complete is to optimize your jobs, builds, workflows, and other tasks to prevent performance degradation, latency, and concurrency. Optimizing your tasks is not simply a function of lessening the workload or number of tasks, but rather, reviewing your optimization and caching strategies to ensure you are meeting your business and technical requirements while also maintaining stability and reliability.
```

Comment:
I agree.

My approach here, would be after getting things working on CircleCI, optimizing comes next because 1) You are a business and decreasing minutes/resources means savings or 2) For anyone, less complex, more efficient builds means saving time.

Reply:
fixed in documentation.